storage/src/main/java/org/apache/uniffle/storage/handler/impl/HdfsShuffleDeleteHandler.java

```diff
@@ -39,33 +38,35 @@ public HdfsShuffleDeleteHandler(Configuration hadoopConf) {
 
   @Override
   public void delete(String[] storageBasePaths, String appId, String user) {
-    Path path = new Path(ShuffleStorageUtils.getFullShuffleDataFolder(storageBasePaths[0], appId));
-    boolean isSuccess = false;
-    int times = 0;
-    int retryMax = 5;
-    long start = System.currentTimeMillis();
-    LOG.info("Try delete shuffle data in HDFS for appId[{}] of user[{}] with {}",appId, user, path);
-    while (!isSuccess && times < retryMax) {
-      try {
-        FileSystem fileSystem = HadoopFilesystemProvider.getFilesystem(user, path, hadoopConf);
-        fileSystem.delete(path, true);
-        isSuccess = true;
-      } catch (Exception e) {
-        times++;
-        LOG.warn("Can't delete shuffle data for appId[" + appId + "] with " + times + " times", e);
+    for (String deletePath : storageBasePaths) {
+      final Path path = new Path(deletePath);
+      boolean isSuccess = false;
+      int times = 0;
+      int retryMax = 5;
```

Comment:
Why do we use `5`?

Reply:
This is the original value. I dont know the reason.